pyproject.toml

```
@@ -76,6 +77,10 @@ tutorial = [
     "notebook",
 ]
 
+imas = [
+    "imas>=4.0.0",
```

where is this dependency used by the code?

---

I see, what is the data-dictionary needed for?

---

The IMAS Data Dictionary describes how IMAS data is organised.  Here though I think you want
```suggestion
    "imas-python>=2.0.0",
```